pyproject.toml

```diff
@@ -1,7 +1,73 @@
 [build-system]
-requires = ["setuptools >= 40.6.0", "wheel"]
+requires = ["setuptools >= 75.3.2", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "django-money"
+dynamic = ["version"]
+description = "Adds support for using money and currency fields in django models and forms. Uses py-moneyed as the money implementation."
+readme = "README.rst"
+license = { text = "BSD" }
+requires-python = ">=3.8"
+maintainers = [
+    { name = "Greg Reinbach", email = "greg@reinbach.com" },
```

Comment:
Even though I am barely active here, I think this list should be updated to reflect the current state of affairs with maintenance

Reply:
Happy to add more maintainers here if you leave a suggestion 👍🏻

Reply:
Well, I think that @benjaoming and I (name = "Dmitry Dygalo", email = "dmitry at dygalo.dev") could be there, if there are no objections :)

Reply:
Yes, I think it's us two!

Benjamin Bach benjamin at overtag.dk

And I think that anyone with experience or specific visions to drive the project is always welcome to contact us :wink:

Reply:
You've already added me, nice! Thanks!